team_config/example_team/component_config.yaml

```diff
@@ -1,22 +1,64 @@
 wamv_camera:
-    - name: front_camera
+    - name: front_left_camera
+      visualize: False
       x: 0.75
-      y: 0.0
+      y: 0.1
+      z: 1.5
+      R: 0.0
+      P: ${radians(15)}
+      Y: 0.0
+      post_Y: 0.0
+    - name: front_right_camera
+      visualize: False
+      x: 0.75
+      y: -0.1
+      z: 1.5
+      R: 0.0
+      P: ${radians(15)}
+      Y: 0.0
+      post_Y: 0.0
+    - name: far_left_camera
+      visualize: False
+      x: 0.75
+      y: 0.3
+      z: 1.5
+      R: 0.0
       P: ${radians(15)}
+      Y: 0.0
+      post_Y: 0.0
 wamv_gps:
     - name: gps_wamv
       x: -0.85
+      y: 0.0
+      z: 1.3
+      R: 0.0
+      P: 0.0
+      Y: 0.0
 wamv_imu:
     - name: imu_wamv
+      x: 0.3
       y: -0.2
+      z: 1.3
+      R: 0.0
+      P: 0.0
+      Y: 0.0
 lidar:
     - name: lidar_wamv
       type: 16_beam
+      x: 0.7
+      y: 0.0
+      z: 1.8
+      R: 0.0
       P: ${radians(8)}
+      Y: 0.0
+      post_Y: 0.0
 wamv_ball_shooter:
     - name: ball_shooter
-      x: 0.5
-      y: -0.25
+      x: 0.55 
+      y: -0.3 
```

Comment:
No particular problem with this, but could you say what you're matching it to? I was looking at the arguments in `wamv_gazebo.urdf.xacro` and it's close but a little different.

Reply:
I was matching the ball shooter coordinates with the ones in [`example_component_config.yaml`](https://github.com/osrf/vrx/blob/master/vrx_gazebo/config/wamv_config/example_component_config.yaml) of the [`vrx_gazebo`](https://github.com/osrf/vrx/tree/master/vrx_gazebo) package so that both the component configurations can be exactly same.